core/env.js

```diff
@@ -85,14 +85,21 @@
   
   /** Get a list of boards that we know about */
   function getBoardList(callback) {
-    Espruino.Core.Utils.getJSONURL(JSON_DIR + "boards.json", function(boards){
+    var jsonDir = Espruino.Config.BOARD_JSON_URL || DEFAULT_JSON_DIR;
```

Comment:
This is for cases when flash menu was opened with a disconnected board. So available boards will be listed from an URL set in settings menu. Am I misunderstand something?